simulator_service/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package>
+  <name>simulator_service</name>
+  <version>0.1.0</version>
+  <description>The simulator_service package</description>
+  <maintainer email="jonathan.meyer@swri.org">Jonathan Meyer</maintainer>
+
+  <license>Apache2</license>
+
+  <build_depend>message_generation</build_depend>
+  <run_depend>message_runtime</run_depend>
+  
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>industrial_robot_simulator</build_depend>
```

Comment:
The `robot_simulator` is a Python package, so I don't think it would be a `build_depend` of this package.